src/components/gui/gui.css

```diff
@@ -34,7 +34,17 @@
 }
 
 .editor-wrapper {
-    flex-basis: 600px;
+    /*
+        This is carefully balanced--  the minimum width at which the GUI will be displayed is 1024px.
+        At that size, the stage pane is 408px wide, with $space of padding to each side.
+        However, we must also add the border width to the stage pane. All-in-all, the stage pane's final width is
+        408px + ($space + $stage-standard-border-width * 2) (one border & padding per left/right side).
+
+        @todo This is in place to prevent "doubling up" of horizontal scrollbars in narrow windows, but there are likely
+        much better ways to solve that (e.g. undo #2124, remove this flex-basis entirely). However, they run their own
+        risks of breaking things, so let's just leave this as-is for the time being.
```

Comment:
Okay, I think I understand all the layout constants now.

This is still reducing tech debt-- hardcoding this to 600px (as was previously done) assumes that `$space` is 8px.

We need the width of this (the editor wrapper) plus the width of the stage pane wrapper to equal 1024px, as that's the app's minimum width (below that, it scrolls).

The stage is 408px wide in "smallish stage mode", used when the window is narrow enough for this to matter, and `$space` is 8px (per side, so 16px in total), so the total space taken up by the stage pane was 424px. Subtract that from 1024px and we arrive at our previous hardcoded 600px figure.

However, the stage pane is now slightly wider because it properly accounts for the stage's border width (426px). This was causing the app wrapper's contents to be 1026px, resulting in a scrollbar in the `.gui` element (scrolling a 1024px viewport of a 1026px-wide area) and a scrollbar in the app wrapper (scrolling a browser-sized viewport of the 1024px-wide `.gui` element).

The new calculation accounts for this, and will always make the editor wrapper + stage pane's widths add up to 1024px.